apps/journeys-admin/src/components/Editor/ControlPanel/Attributes/blocks/Footer/HostSidePanel/HostSidePanel.tsx

```diff
@@ -60,22 +89,39 @@ export function HostSidePanel(): ReactElement {
   }
 
   const handleSelectHost = async (hostId: string): Promise<void> => {
-    setSelectedHost(data.hosts.find((host) => host.id === hostId))
+    setSelectedHost(teamHosts.hosts.find((host) => host.id === hostId))
     setOpenSelect(false)
   }
 
   return (
     <>
       {/* DefaultHostPanel - no host */}
       <SidePanel title={t('Hosted By')} open edit>
-        {!openSelect && selectedHost == null && (
-          <SidePanelContainer border={false}>
-            <ContainedIconButton
-              label={t('Select a Host')}
-              thumbnailIcon={<UserProfiles2Icon />}
-              onClick={() => setOpenSelect(true)}
-            />
-          </SidePanelContainer>
+        {!openSelect && (selectedHost == null || !userInTeam) && (
+          <>
+            <SidePanelContainer border={!userInTeam}>
+              <ContainedIconButton
+                label={t('Select a Host')}
+                disabled={!userInTeam}
+                thumbnailIcon={<UserProfiles2Icon />}
+                onClick={() => setOpenSelect(true)}
+              />
+            </SidePanelContainer>
+            {!userInTeam && team != null && (
+              <SidePanelContainer>
+                <Stack direction="row" alignItems="center" gap={3}>
+                  <AlertCircleIcon />
+                  <Typography variant="subtitle2">
+                    {data?.userTeams.length === 0
+                      ? t('This old journey cannot edit hosts')
```

Comment:
This sentence reads kinda weird lol
```suggestion
                      ? t('Cannot edit hosts for this old journey')
```